src/google-drive/drive.js

```diff
@@ -0,0 +1,162 @@
+    sheets.spreadsheets.values.update({
+      auth: auth,
+      spreadsheetId: spreadsheets_id,
+      range: 'Sheet1!B3:B27',
+      valueInputOption: 'USER_ENTERED',
+      resource: 
+          {range: 'Sheet1!B3:B27',
+          majorDimension: 'COLUMNS',
+          values: timeValues}
+    }, function (err, spreadsheets) {
+      if (err) {
+        // Handle error
+        console.error(err);
+      } else {
+        console.log('Success Hours Column Created');
+      }
+    });
+
+    //For the Days Row
+    sheets.spreadsheets.values.update({
+      auth: auth,
+      spreadsheetId: spreadsheets_id,
+      range: 'Sheet1!C3:I3',
+      valueInputOption: 'USER_ENTERED',
+      resource: 
+          {range: 'Sheet1!C3:I3',
+          majorDimension: 'ROWS',
+          values: dayValues}
+    }, function (err, spreadsheets) {
+      if (err) {
+        // Handle error
+        console.error(err);
+      } else {
+        console.log('Success: Days Row created');
+      }
+    });
+  },
+
+  addTimeValuesToCell: function(auth, values, spreadsheets_id){
+    var sheets = google.sheets('v4');
+    
+    sheets.spreadsheets.values.update({
+      auth: auth,
+      spreadsheetId: spreadsheets_id,
+      range: 'Sheet1!C4:I27',
+      valueInputOption: 'USER_ENTERED',
+      resource: {range: 'Sheet1!C4:I27',
+          majorDimension: 'COLUMNS',
+          values: values}
+    }, function (err, spreadsheets) {
+      if (err) {
+        // Handle error
+        console.error(err);
+      } else {
+        console.log('Success: ', spreadsheets.spreadsheetId);
+      }
+    });
+  },
+
+  addNewRow: function(auth, values, spreadsheetId){
+    var sheets = google.sheets('v4');
+    const request = {
+        auth: auth,
+        spreadsheetId: spreadsheetId,
+        range: "Sheet1!A1:C1",
+        valueInputOption: 'USER_ENTERED',
+        insertDataOption: 'INSERT_ROWS',
+        resource: {
+          values:  [[new Date().toISOString(),'Brian', 'www.facebook.com']]
+        }
+      };
+    sheets.spreadsheets.values.append(
+      request,function (err, spreadsheets) {
+        if (err) {
+          // Handle error
+          console.error(err);
+        } else {
+          console.log('Success: Added Row To', spreadsheets.spreadsheetId);
+          console.log('Rows Modified are: ', spreadsheets.tableRange);
+        }
+      }
+    );
+
+  },
+
+  createFolder: function(auth) {
+    var fileMetadata = {
+      'name': 'Invoices',
+      'mimeType': 'application/vnd.google-apps.folder'
+    };
+    var drive = google.drive({ version: 'v3', auth: auth });
+    drive.files.create({
+      resource: fileMetadata,
+      fields: 'id'
+    }, function (err, file) {
+      if (err) {
+        // Handle error
+        console.error(err);
+      } else {
+        console.log('Folder Id: ', file.id);
+      }
+    });
+  },
+
+//returns assignment ID
+  createAssignment: function(assignementName) {
+    authentication.authenticate().then((auth)=>{
```

Comment:
You need a `return` on this line. That way this function will return a Promise a you'll be able to successfully await in  [here](https://github.com/teach-bot/teach-bot/pull/6/files#diff-1649518ca8af8692b8ff75c74087dd19R19).